packages/local_contexts/local_contexts.js

```diff
@@ -0,0 +1,169 @@
+var cvoc_lc_projectSelector = "span[data-cvoc-protocol='localcontexts']";
+var cvoc_lc_projectInputSelector = "input[data-cvoc-protocol='localcontexts']";
+
+var cvoc_lc_lcBaseUrl = "https://localcontextshub.org";
+
+var cvoc_lc_seach_minimumInputLength = 4;
+var cvoc_lc_search_delay = 1000;
```

Comment:
This makes it seem a bit slow. Could it be faster, e.g. 300-500 ms, or is this required by LC?

Reply:
their api is sometimes not that fast, so I didn't want to overwhelm it :). but I brought it to 500ms now.